README.md

```diff
@@ -23,7 +23,7 @@
         <th><a href="../../tree/development"><code>development</code></a></th>
     </tr>
     <tr>
-        <td>Ubuntu</td>
+        <td>Ubuntu Latest (24.04)</td>
```

Comment:
Please undo the changes in the README. We can just test for multiple OS versions within the same workflow and group it under Linux.